documentation/modules/exploit/windows/rdp/cve_2019_0708_bluekeep_rce.md

```diff
@@ -15,20 +20,13 @@ pool base in kernel memory and set it as the `GROOMBASE` option.
 
 ## Vulnerable Application
 
-This exploit should work against a vulnerable RDP service from one of these Windows systems:
-
-* Windows 2000 x86 (All Service Packs))
-* Windows XP x86 (All Service Packs))
-* Windows 2003 x86 (All Service Packs))
-* Windows 7 x86 (All Service Packs))
-* Windows 7 x64 (All Service Packs)
-* Windows 2008 R2 x64 (All Service Packs)
-
 This exploit module currently targets these Windows systems running on several virtualized and physical targets.
 
 * Windows 7 x64 (All Service Packs)
```

Comment:
The targets section only lists SP1?
```
Available targets:
  Id  Name
  --  ----
  0   Automatic targeting via fingerprinting
  1   Windows 7 SP1 / 2008 R2 (6.1.7601 x64)
  2   Windows 7 SP1 / 2008 R2 (6.1.7601 x64 - Virtualbox 6)
  3   Windows 7 SP1 / 2008 R2 (6.1.7601 x64 - VMWare 14)
  4   Windows 7 SP1 / 2008 R2 (6.1.7601 x64 - VMWare 15)
  5   Windows 7 SP1 / 2008 R2 (6.1.7601 x64 - VMWare 15.1)
  6   Windows 7 SP1 / 2008 R2 (6.1.7601 x64 - Hyper-V)
  7   Windows 7 SP1 / 2008 R2 (6.1.7601 x64 - AWS)

```

Reply:
Great! Thank you. That is correct.

Reply:
I'm not sure about 2008, so I'm taking out "all service packs" to be conservative.